linux/keyman-config/run-tests.sh

```diff
@@ -14,7 +14,15 @@ fi
 
 if [[ -n "${TEAMCITY_VERSION}" ]]; then
   if ! pip3 list --format=columns | grep -q teamcity-messages; then
-      pip3 install teamcity-messages
+      if [[ -n "${TEAMCITY_PLATFORM}" ]] || [[ -n "${DOCKER_RUNNING}" ]]; then
```

Comment:
Ha! You are testing teamcity in a different way here than in #13810! And I am not sure about the interaction here in l.17 with l.15?

Reply:
Ah, you're right. I changed the outer one and added a (hopefully) clarifying comment why we use two different `TEAMCITY_*` variables in l15 and l17.